lib/zendesk_apps_support/package.rb

```diff
@@ -33,15 +34,12 @@ def validate(marketplace: true)
         if has_manifest?
           errors << Validations::Source.call(self)
           errors << Validations::Translations.call(self)
+          errors << Validations::Requirements.call(self)
 
-          unless manifest.requirements_only?
+          if !manifest.requirements_only? && manifest.marketing_only?
```

Comment:
marketing only don't have templates and stylesheets either, this looks like it will run though